core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -161,6 +163,18 @@ default ConnectorTableHandle makeCompatiblePartitioning(ConnectorSession session
         return Optional.empty();
     }
 
+    /**
+     * Return the column handles for the columns that must be present to in order
+     * to perform the partitioning and/or bucketing required.  By default, the table
+     * has no such columns.
+     * @return Return a list of the column handles required for partitioning and/or
+     * bucketing for rows in the table.
```

Comment:
That's redundant over the paragraph above

Reply:
Changed to "@return Return the write redistribution columns for the table"